packages/app/src/components/RenderDataValue.tsx

```diff
@@ -205,7 +205,14 @@ const scalarRenderers: {
       </div>
     );
   },
-  binary: ({ value }) => <>Binary (length {value.value.length.toLocaleString()})</>,
+  binary: ({ value }) => {
+    // FIXME: Coercing `value.value` into a `Uint8Array` here because `Uint8Array` gets parsed as an
+    //        object of shape `{ [index: number]: number }` when stringified via `JSON.stringify()`.
+    //        Consider coercing it back to `Uint8Array` at the entrypoints of the boundaries between
+    //        browser and node.js instead.
+    const coercedValue = new Uint8Array(Object.values(value.value));
+    return <>Binary (length {coercedValue.length.toLocaleString()})</>;
+  },
```

Comment:
This fixes an issue where hovering on the new `binary` output port causes the app to crash:

![CleanShot 2024-04-27 at 19 50 15@2x](https://github.com/Ironclad/rivet/assets/3143132/a70131d6-70f7-4f57-b31c-5eff57fe27c9)

It crashes because `value.value` is a POJO instead of a `Uint8Array`. For example, `Uint8Array(2) [123, 45]` would end up being `{ "0": 123, "1": 45 }` here.

I suspect this is because `Uint8Array` is serialised as such a POJO when stringified with `JSON.stringify()`—as is required when passing the data over to the Node.js executor. This can be verified with:

```tsx
const binaryOnBrowser = new Uint8Array([123,45]); // Uint8Array(2) [123, 45]
const binaryInTransit = JSON.stringify(binaryOnBrowser);
const binaryOnNode = JSON.parse(binaryInTransit); // { "0": 123, "1": 45 }
```

There's probably a better place to put this coercion fix but I'm not sure where. 😬 Need the help of the maintainers on this one.